### PR TITLE
Minor fixes/improvements

### PR DESCRIPTION
**Pull request by BaileyLawson**

### CODE REVIEW

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[708af13](https://github.com/BaileyLawson/PowerShell-Docs/pull/3899/commits/708af130e3aa66789dbda4c37847964c283b9365)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md](https://github.com/BaileyLawson/PowerShell-Docs/blob/patch-1/reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-6&branch=pr-en-us-3899) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Functions_Advanced_Parameters.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Functions_Advanced_Parameters.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md](https://github.com/BaileyLawson/PowerShell-Docs/blob/patch-1/reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C13%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903131244482473-3899%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=gzKtbpvMAQYwuP%2Bz1iY5wHrgjdidpJgFD3mEgou5ztM%3D&st=2019-03-13T12%3A44%3A09Z&se=2019-04-13T12%3A49%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[dce3ce2](https://github.com/BaileyLawson/PowerShell-Docs/pull/3899/commits/dce3ce2b9d7fb1184c790830bf90a7f966adcea1)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md](https://github.com/BaileyLawson/PowerShell-Docs/blob/patch-1/reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-6&branch=pr-en-us-3899) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Functions_Advanced_Parameters.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Functions_Advanced_Parameters.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md](https://github.com/BaileyLawson/PowerShell-Docs/blob/patch-1/reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C13%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903131246313332-3899%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=RMBItih6n3P8%2BxXLI60tXJoxTuEhaDkgGstE6xRN3cg%3D&st=2019-03-13T12%3A48%3A20Z&se=2019-04-13T12%3A53%3A20Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md [hunk 1]
```diff
@@ -39,10 +39,10 @@ has the following characteristics:
 - It takes input from the pipeline.
 - It takes an array of strings as input.
 
-```powershell
+```PowerShell
```
Comment (sdwheeler) [inline reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md]: Do not camel-case code fence tags. Please fix all instances.

Comment (cblackuk) [reply reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md]: It is Pascal case and also everywhere in Azure and Azure Stack docs we do it that way...

https://raw.githubusercontent.com/MicrosoftDocs/azure-docs/master/articles/azure-stack/azure-stack-powershell-install.md

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[5e2b9dc](https://github.com/BaileyLawson/PowerShell-Docs/pull/3899/commits/5e2b9dc8a54e69806274d19c8c852ac34bf5ef8c)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md](https://github.com/BaileyLawson/PowerShell-Docs/blob/patch-1/reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-6&branch=pr-en-us-3899) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Functions_Advanced_Parameters.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Functions_Advanced_Parameters.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md](https://github.com/BaileyLawson/PowerShell-Docs/blob/patch-1/reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C13%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903131736169581-3899%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=k5Gx012dNzgj1f5We4XzsFdESBf9UKslBu96UNJ0hXs%3D&st=2019-03-13T17%3A36%3A05Z&se=2019-04-13T17%3A41%3A05Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by cblackuk** [comment]
> Do not camel-case code fence tags. Please fix all instances.

I Microsoft docs repo you have 974 times "```PowerShell" for instance... so it is pretty unfair to be asking to change it from Pascal case here but sure :-) Maybe we need a wider PR to bring consistency to all MS Docs?

**Comment by zjalexander** [comment]
OPS Build status updates of commit _[e5f51ae](https://github.com/BaileyLawson/PowerShell-Docs/pull/3899/commits/e5f51ae11b666019041eee8ff54590741a5ecfc8)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md](https://github.com/BaileyLawson/PowerShell-Docs/blob/patch-1/reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/PowerShell/module/microsoft.powershell.core/about/about_functions_advanced_parameters?view=powershell-6&branch=pr-en-us-3899) (powershell-6) | [Details](#user-content-reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Functions_Advanced_Parameters.md)

<a name="reference%2F6%2FMicrosoft.PowerShell.Core%2FAbout%2Fabout_Functions_Advanced_Parameters.md"></a>
### [reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md](https://github.com/BaileyLawson/PowerShell-Docs/blob/patch-1/reference/6/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md)
 - **[Warning]** Metadata with following name(s) are not allowed to be set in Yaml header, or as file level metadata in docfx.json, or as global metadata in docfx.json: locale. They are generated by OPS, so the values set in these 3 places will be ignored. Please remove them from all 3 places to resolve the warning.

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C3%5C13%5C6ff7e8c3-dfc6-3ebd-da5a-d5e2ff43de8f%5CPullRequest%5C201903131745112251-3899%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Kcr3eYlQPODhRKG80Xk3UZ0WLDO%2FUQ8wLewV6ssbrmk%3D&st=2019-03-13T17%3A44%3A32Z&se=2019-04-13T17%3A49%3A32Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**